specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -279,6 +279,223 @@ interface DevBoxes {
+      @body
+      body: CustomizationGroup;
+    },
+    CustomizationGroup
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "No model for this."
+  @doc("Gets the log for a customization task.")
+  @route("/projects/{projectName}/users/{userId}/devboxes/{devBoxName}/customizationGroups/{customizationGroupName}/logs/{customizationTaskId}")
+  @get
+  @added(APIVersions.v2024_05_01_preview)
+  getCustomizationTaskLog is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+
+      @doc("""
+        The AAD object id of the user. If value is 'me', the identity is taken from the 
+        authentication context.
+      """)
+      @maxLength(36)
+      @minLength(2)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$|^me$")
+      @path
+      userId: string;
+
+      @doc("The name of a Dev Box.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      devBoxName: string;
+
+      @doc("A customization group name.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      customizationGroupName: string;
+
+      @doc("A customization task ID.")
+      @maxLength(36)
+      @minLength(36)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$")
+      @path
+      customizationTaskId: string;
+
+      @doc("Accept header.")
+      @header
+      accept: "text/plain";
+    },
+    {
+      @statusCode
+      statusCode: 200;
+
+      @body
+      body: string;
+    }
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard operations produce route with /catalogs, but we only allow list on /projects"
+  @doc("Lists all customization tasks available to the project.")
+  @route("/projects/{projectName}/customizationTasks")
+  @get
+  @added(APIVersions.v2024_05_01_preview)
+  listCustomizationTaskDefinitionsByProject is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+    },
+    CustomizationTaskDefinitionResult
+  >;
+
+  @doc("Gets a customization task.")
+  @added(APIVersions.v2024_05_01_preview)
+  getCustomizationTaskDefinitions is StandardResourceOperations.ResourceRead<CustomizationTaskDefinition>;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "StandardOperations produces route on an specific customizationTask, but we want it on the group"
+  @doc("Validates a list of customization tasks.")
+  @action("validateGroup")
+  @route("/projects/{projectName}/customizationTasks:validateGroup")
```

Comment:
I believe if you add the @actionSeparator decorator here as well then you should be able to remove the action portion of the route as it will get add automatically with the decorator.

Reply:
https://typespec.io/docs/libraries/rest/reference/decorators#@TypeSpec.Rest.actionSeparator

Reply:
Doesn't seem to work here unfortunately, if the `@route()` attribute is added, it supersedes everything else, so this would only work without `@route`. But I need to add it since it's a custom `Azure.Core.Foundations.Operation` operation